src/main/java/org/jabref/logic/importer/fetcher/CompositeSearchBasedFetcher.java

```diff
@@ -18,7 +18,7 @@
 
 public class CompositeSearchBasedFetcher implements SearchBasedFetcher {
 
-    public static final String FETCHER_NAME = "Search pre-configured";
```

Comment:
Why do you revert it?

Reply:
Not doing so made the dropdown list display "Search Selected" (the old value) instead of "Search pre-configured"

The dropdown in question (from the latest revert):
![image](https://github.com/user-attachments/assets/e6f4ec6d-1575-45b2-92b3-9fa67b7f949b)

Reply:
Though if it was with the revert before that, I was trying to look into the errors that showed up. Though I guess they were quite moot now that I know they're optional